scripts/aws/deploy_repo_to_prod_env_api_gw.sh

```diff
@@ -8,7 +8,7 @@
 
 python3 ./scripts/aws/deploy.py \
   --tag  None \
-  --api-base-domain api.edx.org  \
+  --api-base-domain studio.api.edx.org  \
```

Comment:
I don't think it's good to change the url here. This repo will then only be usable for the studio api, unless you change the code again, and also maybe someone mistakenly deploys without correcting the url.

Or is this file only for studio api deployment?

It could be better to have this url as an argument - you can specify it in the invocation_history/ scripts - or something like that.

Reply:
Good point. I should leave `deploy.py`'s usage as is, in case there are existing scripts referencing it outside our organization, and create. But I can create a new, parameterized script and change `deploy.py`'s implementation to delegate to that parameterized script.

Reply:
You're right that something is afoul here, but it's even more than what you've called out. Having abandoned use of the omnibus API for studio, we no longer want most of those stage variables set when working with the Studio API. I'll rename the script to make it clear that it's only to be used with the Studio API, and clean up the stage variables.